src/redux/rockets/rockets.js

```diff
@@ -0,0 +1,47 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+const ALL_ROCKETS = 'space travelers/redux/rockets/ALL_ROCKETS';
+const RESERVE_ROCKET = 'space travelers/redux/rockets/RESERVE_ROCKET';
+
+export const rocketsReducer = (state = { status: 'default', rockets: [] }, action) => {
+  switch (action.type) {
+    case `${ALL_ROCKETS}/fulfilled`:
+      // console.log(action.payload);
```

Comment:
- [ Required ] Please consider removing any piece of commented code from your codebase, commented code makes your project look unprofessional. Kindly rectify this.

Reply:
Hi @Felix45 

Thank you for noticing that, we forgot to remove it.